src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  height: 100%;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+  padding: 32px 16px 16px;
+  font-family: $main-font-family;
+  color: $main-color;
+
+  &__photo {
+    width: 100%;
+    height: 134px;
+    background-size: contain;
+    background-repeat: no-repeat;
+    margin: 0 3px 40px;
```

Comment:
The horizontal margin might not be necessary here. If you want to center the image, you could use `margin: 0 auto;` instead.